### ISSUE TITLE
feature: "copy"/"paste" node in Graph Editor

### ISSUE THREAD

**Issue by whatisgalen** [archesproject/arches#10528]
There are multiple scenarios in which it would be useful to be able to copy a node and paste it into a different graph (or a different place on the same graph). For example, say you chose concept as the datatype but want to change it to concept-list. It must be deleted and recreated; being able to copy everything about the node and paste it into the new one would save time. 

How to implement? One way would be to copy some json capturing all the properties of the node config and the relevant properties of its parent. "Pasting" would have to be handled by some "import node" logic that checks if the ontology property of the old parent matches that of the target parent, if not leaves it blank. It would also check if the necessary card and widget are present in the target db and if not, falls back on whatever the defaults are.

In short, while this would really be an "import"/"export" node spec, it could be seamless enough to feel like copy/paste.

**Comment by chiatt** [comment]
Why not just create a branch?

**Comment by whatisgalen** [comment]
> Why not just create a branch?

I think creating a branch implies you already know the config for a node a priori. The example I mentioned is where, lets say you've imported a graph from somewhere else but there's a bunch of changes you want to make, now what? There isn't really a branch functionality to help in that case.

**Comment by chiatt** [comment]
I believe being able to change the datatype of a node is something that gets resolved in v8. @chrabyrd could probably confirm that.

**Comment by chrabyrd** [comment]
👋 ! I don't believe changing a node's datatype is slotted for v8. I remember attempting to do this last year but the interactions with ES were too opaque to allow for it. But anything's possible and if I end up with extra bandwidth I could give it another look

**Comment by bferguso** [comment]
Also - just to add to this - the Export Branch -> Import Branch strategy, which I've been currently using requires manual editing of the Graph JSON if the branch used to copy the sub-tree isn't kept (see issue #9970).
